<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="firstEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter text:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="firstConfirmButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="firstBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="secondEdit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="secondBrowser">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter massive and its size:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="massiveSpinBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="secondConfirmButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
